game-core/src/main/java/org/triplea/debug/TextUtils.java

```diff
@@ -7,7 +7,7 @@ private TextUtils() {}
 
   static String textToHtml(final String text) {
     return "<html>"
-        + HtmlEscapers.htmlEscaper().escape(text).replaceAll(System.lineSeparator(), "<br/>")
+        + HtmlEscapers.htmlEscaper().escape(text).replaceAll("\r?\n", "<br/>")
```

Comment:
Are you sure we need `\r`? AFAIK it's not actually needed in most places where we have it. I think we have it in so many places due to confusion vs actually needing it.
Isn't `lineSeperator()` also preferred? I'm curious how this breaks.

Reply:
I don't know if it's actually needed, but in case it is it will be replaced as well.
The reason why this test didn't work is the following:
The test case passed some string along the lines of `Test\nString`, where on linux the `\n` got replaced with `<br>`, but on linux the test was trying to replace `\r\n` in the string which obviously isn't part of it, so the string stayed the same.

That's why the `\r` is optional, it would probably still work if it got ommited, but \r is really uneccesary